mmv1/third_party/terraform/website/docs/d/project.html.markdown

```diff
@@ -30,6 +30,7 @@ The following arguments are supported:
 
 * `project_id` - (Optional) The project ID. If it is not provided, the provider project is used.
 
+* `number` - The numeric identifier of the project.
```

Comment:
This field is a computed field, so it should go under the `The following attributes are exported:` line

Reply:
done